There are some potential issues when running the validate.py

# --ignore CountValidator

Please use residential proxy to bypass protection.
